### PR TITLE
Fix some typos, backticks, new clippy lint.

### PR DESCRIPTION
**Pull request by waywardmonkeys**

### CODE REVIEW

**Comment by waywardmonkeys** [comment]
And yep, I already submitted the actual typos upstream.

File: vello_encoding/src/math.rs [hunk 1]
```diff
@@ -122,7 +123,8 @@ pub(crate) fn f32_to_f16(val: f32) -> u16 {
     output | (sign >> 16) as u16
 }
 
-/// Convertes a 16-bit precision IEEE-754 binary16 float to a f32.
+/// Converts a 16-bit precision IEEE-754 binary16 float to a `f32`.
+///
 /// This implementation was adapted from Fabian Giesen's `half_to_float()`
 /// function which can be found at <https://gist.github.com/rygorous/2156668#file-gistfile1-cpp-L574>
 pub fn f16_to_f32(bits: u16) -> f32 {
```
Comment (DJMcNab) [inline vello_encoding/src/math.rs:130]: The final three items in this module paint a funny story:
- `#[allow(dead_code)] fn point_to_f32`
- `pub(crate) fn f32_to_f16`
- `pub fn f16_to_f32` (i.e. the inverse of the `pub(crate)` item above)

**Review by DJMcNab — APPROVED** [review]

File: vello_encoding/src/math.rs (hunk 1 above)
Comment (waywardmonkeys) [reply vello_encoding/src/math.rs:130]: And it is a public mod so there's a chance that skia is actually. using something. Who knows!

Comment (waywardmonkeys) [reply vello_encoding/src/math.rs:130]: And that `allow(dead_code)` should be a `full` feature gate.

**Pull request merged**